vertx-config-consul/src/main/java/examples/ConfigConsulExamples.java

```diff
@@ -36,6 +36,7 @@ public void example1(Vertx vertx) {
           .put("host", "localhost")
           .put("port", 8500)
           .put("prefix", "foo")
+          .put("raw-data", false)
```

Comment:
is that needed if false is the default value ?

Reply:
Default value is true. But regardless of the default value, all possible options should be shown in the example.